src/ensembl/src/shared/accordion/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+type AccordionProps = React.HTMLAttributes<HTMLDivElement>;
+
+const Accordion = ({ ...rest }: AccordionProps): JSX.Element => {
```

Comment:
I am curious why you didn't use a param for this rather than rest operator? You could write this as (props: AccordionProps) and use it as a spread operator in the `div` element as you've already done with ...rest.

Reply:
Well spotted! I couldn't think of any reason why `...rest` will be required here instead of props. I've updated it now.